src/utils/DockerUtil.js

```diff
@@ -274,6 +311,15 @@ export default {
     });
   },
 
+  status (name) {
+    let running = false;
+    let container = this.client.getContainer(name);
```

Comment:
I don't think this actually returns a container, but rather a reference you can use to call `inspect`, `restart`, etc.

E.g.

```
let container = this.client.getContainer(name);
container.inspect(err, data => {});
```

It seems that this method would return false every time. Is it needed?